### ISSUE TITLE
Stuck in dialogue during Overdesigned

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#89]
During the final step of the mission Overdesigned in which you talk to Walter Stroud, you will be trapped in the dialogue zoom in after he finishes speaking. It works exactly how it does during the same glitch at the end of All That Money Can Buy, except for some reason the kill command will not knock him out of dialogue this time. Therefore turning in this quest is impossible.

---

## Details 
Walter Stroud: 00005998
Overdesigned: MS05 (00299E79)
The latest done stage is 800, while setting stage to 1000 will bypass talking to Walter.

## Game Version
1.7.23.0

## New Game
No

## Reproduction Steps
During the quest Overdesigned, go for the compromising approach where each member of the team cuts one of their planned design pieces. I also only did the bounty and not the transport step during the info gathering phase. Follow the quest through normally and go back to Walter to inform him of the job being finished, he will give credits and tell you to pick up your new ship, and the dialogue zoom will never go away.

**Comment by Constellation-VASCO** [comment]
A possible temporary fix for this is that before turning the quest, get out of the lodge, go to a named NPC, like that scientist near the tree >> open console command >> click on the npc >> close command >> go to Walter and turn the quest. Once you're locked into the conversation, open the command again and do the player moveto (write down the npcs's ref id) so you'd get teleported outside of the lodge.

I managed to get out of the conversation this way but only to find out that I got hit with another bug, no ship reward from completing the quest.
